SWIPs/swip-data-types-descriptions.md

```diff
@@ -0,0 +1,66 @@
+---
+SWIP: <to be assigned>
+title: Swarm node implementer spec - 
+author: Louis Holbrook @nolash <dev@holbrook.no>
+status: Draft
+type: Track Specs
+created: 2019-08-02
+---
+
+## Abstract
+
+This SWIP is a part of a general specification of a Swarm node. The SWIP system is used for convenience only.
+
+## Motivation
+
+Enable other implementations of the swarm node.
+
+## Backwards Compatibility
+
+N/A
+
+## Test Cases
+
+N/A
+
+## Implementation
+
+N/A
+
+## Copyright
+
+Copyright and related rights waived via CC0
+
+## Specification
+
+### data representations
+
+This document uses the `ABNF` convention for defining data types, as
+described in the IETF RFC 5234. 
+
+#### primitives
+
+The basic identifiers are defined as follows:
+
+    UINT64MAX   = 18446744073709551615
```

Comment:
table format

Reply:
I've changed it to table, but I must admit I prefer the classic ABNF definition syntax. I also thought we agreed on keeping this representation for the ABNF, but instead tables for other definition lists.

One reason to avoid tables in this case is the superfluous header column that is required for md tables.

Reply:
hm alright, forgot :sweat: different schools perhaps?

>  superfluous header column

usually the header column is meaningful, what's an example of where it isn't?

cc @decanus